recipes/rpclib/all/conanfile.py

```diff
@@ -1,87 +1,110 @@
-from conans import ConanFile, tools, CMake
-from conan.tools.microsoft import msvc_runtime_flag
 import os
 
-required_conan_version = ">=1.43.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, export_conandata_patches, get, rmdir, rename, mkdir
+from conan.tools.microsoft import is_msvc_static_runtime
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
 
 
 class rpclibConan(ConanFile):
     name = "rpclib"
     description = "A modern C++ msgpack-RPC server and client library."
     license = "MIT"
-    topics = ("rpc", "ipc", "rpc-server")
-    homepage = "https://github.com/rpclib/rpclib/"
     url = "https://github.com/conan-io/conan-center-index"
-    exports_sources = ["CMakeLists.txt", "patches/*"]
-    generators = "cmake"
+    homepage = "https://github.com/rpclib/rpclib/"
+    topics = ("rpc", "ipc", "rpc-server")
+
+    package_type = "library"
     settings = "os", "arch", "compiler", "build_type"
     options = {
         "shared": [True, False],
-        "fPIC": [True, False]
+        "fPIC": [True, False],
     }
     default_options = {
         "shared": False,
-        "fPIC": True
+        "fPIC": True,
     }
-    _cmake = None
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 14
 
     @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "msvc": "14.1",
```

Comment:
it's not the correct msvc versioning scheme of conan. Fixed in https://github.com/conan-io/conan-center-index/pull/19916

Reply:
I'm pretty sure these versions are too strict and its builds with lower versions, since there was no checks prior to this PR and c3i was testing Visual Studio 2015 and gcc 4.9